themes/digital.gov/layouts/communities/list.html

```diff
@@ -1,123 +1,27 @@
-                      >
-                        Connect with <br />{{ $e.Params.dg_shortname }}
-                      </a>
-                    </div>
-                  </div>
-                </li>
-              {{ end }}
-            </ul>
-          </div>
-        </div>
-      </div>
+      {{ partial "core/featured-communities.html" (dict "context" . "layout" "list") }}
     </section>
 
-    <div class="grid-container grid-container-desktop">
-      <div class="grid-row tablet-lg:grid-gap-4">
-        <div class="grid-col-12">
-          <header class="page-head page-head-{{- .Type -}}">
-            {{/* Page Title */}}
-            <h2>Other Communities of Practice</h2>
-
-            <div class="deck">
-              These communities are not managed by Digital.gov, but are listed
-              here <br />because they overlap with
-              <a class="usa-link" href="{{- "/about/" | relURL -}}"
-                >our mission</a
-              >.
-            </div>
-          </header>
-        </div>
+    {{/* Other Communities of Practice */}}
+    <header class="page-head">
+      <h2>Other Communities of Practice</h2>
+      <div class="deck">
+        These communities are not managed by Digital.gov, but are listed here
+        <br />because they overlap with
+        <a class="usa-link" href="{{- "/about/" | relURL -}}">our mission</a>.
       </div>
-    </div>
+    </header>
 
     <section class="usa-section">
-      <div class="grid-container-desktop">
-        <ul class="communities-of-practice--list usa-collection">
-          {{/* Gets all communities pages */}}
-          {{ $communities := (where .Site.Pages "Section" "communities").ByTitle }}
-
-          {{/* Gather only the pages with a weight greater than or equal to 1 */}}
-          {{ $communities := (where $communities ".Params.weight" "ge" 1) }}
-
-          {{/* Gather only the pages with a dg-emphasis key */}}
-          {{ $communities := (where $communities ".Params.dg_highlight" "!=" true) }}
-
-          {{ range $i, $e := $communities }}
-            <li class="usa-collection__item">
-              <div class="usa-collection__body">
-                <h2 class="usa-collection__heading">
-                  <a
-                    class="usa-link {{ if $e.Params.redirectto }}
-                      usa-link--external
-                    {{ end }}"
-                    href="{{- $e.Permalink -}}"
-                    title="{{- $e.Title | markdownify -}}"
-                  >
-                    {{- $e.Title | markdownify -}}
-                  </a>
-                </h2>
-                <p class="usa-collection__description">
-                  {{- $e.Params.summary | markdownify -}}
-                </p>
-              </div>
-            </li>
-          {{ end }}
-        </ul>
-      </div>
+      {{ partial "core/collection/collection-grid-list.html" (dict "list_data" $other_communities) }}
```

Comment:
Moved this collection related markup to it's own partial in the collection partials folder.